documentation/projects/proposals/rekognition_data/20240510-implementation_plan_machine_generated_tags_frontend.md

```diff
@@ -0,0 +1,199 @@
+   labelling all the tags generated by a single provider with a "by provider"
+   heading, and adding the accuracy to the tag's inner content. This way, the
+   user will see the provider and the accuracy of the tag without any additional
+   actions. As a bonus, if a tag is generated by multiple providers, the user
+   will see that different providers generate the same tag, and will be able to
+   see the difference in accuracy between the providers.
+
+## Design
+
+<!-- Note any design requirements for this plan. -->
+
+The first iteration of the designs that use the first alternative option is
+ready:
+[Displaying machine-generated content](https://github.com/WordPress/openverse/issues/4192)
+If we decide to go with the third alternative option, we will need to add the
+provider headings and the accuracy to the tags.
+
+## Blockers
+
+<!-- What hard blockers exist that prevent further work on this project? -->
+
+There are no blockers for this work.
+
+## Accessibility
+
+<!-- Are there specific accessibility concerns relevant to this plan? Do you expect new UI elements that would need particular care to ensure they're implemented in an accessible way? Consider also low-spec device and slow internet accessibility, if relevant. -->
+
+Adding the "Source tags" and "Generated tags" headings is good for the
+accessibility of the tag list. Both the sighted and the screen reader users will
+understand what the list is for. It is always desirable to have visible label
+next to a list of elements.
+
+The display of generated tags' provider and accuracy is, on the other hand, a
+challenging problem from accessibility standpoint. Tooltips are often described
+as inaccessible for mobile users who do not have a hover action. MDN's section
+on tooltip best practices suggests providing "clear labels and sufficient body
+text" instead of tooltips that hide important information. We could add the tag
+accuracy to the However, if we decide to add the provider to the generated tags,
+there will be a lot of duplication.
+
+## Rollback
+
+<!-- How do we roll back this solution in the event of failure? Are there any steps that can not easily be rolled back? -->
+
+We can roll back this solution by reverting the PRs implementing this.
+
+## Localization
+
+<!-- Any translation or regional requirements? Any differing legal requirements based on user location? -->
+
+The tag headings will be translated as usual. For the "About page" it is
+important to split the text into shorter chunks to make them easier to handle
+for translators. The eslint rule will make sure that this is done properly.
+
+## Risks
+
+<!-- What risks are we taking with this solution? Are there risks that once taken can’t be undone?-->
+
+## Prior art
+
+<!-- Include links to documents and resources that you used when coming up with your solution. Credit people who have contributed to the solution that you wish to acknowledge. -->
```

Comment:
I think if we don't have any information for these sections, we can remove them.